### PR TITLE
Add manual testing for Add a Buyer

### PR DESCRIPTION
**Pull request by zengzihui**
Add manual test cases for add buyer command.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -955,6 +955,39 @@ testers are expected to do more *exploratory* testing.
...
+**Prerequisites:**
+- List all persons using the `list` command.
+- There is currently no `person` with the name "James Cook", "Kris Hua". "John Carl 1" exists as a seller in EstateEase.<br>
+
+1. **Add buyer**
+    1. **Test case** (add a `buyer` named `James Cook`): `addBuyer n/James Cook p/98753432 e/jamescook@example.com budget/550000 type/Hdb`<br>
+       **Expected**: A new buyer is added, with name `James Cook`, phone `98753432`, email `jamescook@example.com`, budget `550000`, preferred housing type `Hdb`<br><br>
+
+2. **Invalid format**
+    1. **Test case** (add a `buyer` without compulsory details): `addBuyer` <br>
```
Comment (felixchanyy) [inline docs/DeveloperGuide.md:970]: Missing colon

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -955,6 +955,39 @@ testers are expected to do more *exploratory* testing.
    2. **Test case** (Duplicate house): `addSeller n/Carl Lim p/98765432 e/johncarl@example.com type/Hdb street/Clementi Ave 2 blk/311 level/02 unitNo/25 postal/578578 price/999999999` <br>
       **Expected**: No seller is added. Error indicating that the house already existed in the data. <br>
 
+[//]: # (@@author zengzihui)
+### Adding a buyer
```
Comment (lokidoki102) [inline docs/DeveloperGuide.md:959]: Change from `Adding a buyer` -> `Adding a Buyer`

Comment (felixchanyy) [inline docs/DeveloperGuide.md:959]: Cap the buyer

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -955,6 +955,39 @@ testers are expected to do more *exploratory* testing.
...
+
+**Prerequisites:**
+- List all persons using the `list` command.
+- There is currently no `person` with the name "James Cook", "Kris Hua". "John Carl 1" exists as a seller in EstateEase.<br>
+
+1. **Add buyer**
+    1. **Test case** (add a `buyer` named `James Cook`): `addBuyer n/James Cook p/98753432 e/jamescook@example.com budget/550000 type/Hdb`<br>
+       **Expected**: A new buyer is added, with name `James Cook`, phone `98753432`, email `jamescook@example.com`, budget `550000`, preferred housing type `Hdb`<br><br>
```
Comment (felixchanyy) [inline docs/DeveloperGuide.md:967]: : before **

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -955,6 +955,39 @@ testers are expected to do more *exploratory* testing.
...
 
+[//]: # (@@author zengzihui)
+### Adding a Buyer
+
+**Prerequisites:**
+- List all persons using the `list` command.
+- There is currently no `person` with the name "James Cook", "Kris Hua". "John Carl 1" exists as a seller in EstateEase.<br>
+
+1. **Add buyer**
+    1. **Test case:** (add a `buyer` named `James Cook`): `addBuyer n/James Cook p/98753432 e/jamescook@example.com budget/550000 type/Hdb`<br>
```
Comment (felixchanyy) [inline docs/DeveloperGuide.md:966]: Perhaps remove the ( ... )

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -955,6 +955,39 @@ testers are expected to do more *exploratory* testing.
...
+
+4. **Invalid Value for certain fields**
+   1. **Test case:** (add a `buyer` with an invalid `Preferred Housing Type`): `addBuyer n/Neo Ng p/98765432 e/neong@examaple.com budget/99999900 type/HDB` <br>
+       **Expected:** No buyer is added. Error indicating HousingType should only be 'Landed', 'Hdb' or 'Condominium'. HousingType is case-sensitive. <br><br>
+   
+   2. **Test case:** (add a `buyer` with an invalid `budget`): `addBuyer n/Grace Tan p/98765432 e/gracetan@examaple.com budget/-99999900 type/Hdb` <br>
+      **Expected:** No buyer is added. Error indicating budget should be a positive number. <br><br>
+   
+   3. **Test case:** (add a `buyer` with an invalid `budget`): `addBuyer n/Chris Ong p/98765432 e/chrisong@examaple.com budget/aa type/Hdb` <br>
+         **Expected:** No buyer is added. Error indicating budget should be a positive number. <br><br>
```
Comment (felixchanyy) [inline docs/DeveloperGuide.md:988]: remove one `<br>`

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (zengzihui) [reply docs/DeveloperGuide.md]: Fixed alr

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (zengzihui) [reply docs/DeveloperGuide.md]: Updated already

Comment (zengzihui) [reply docs/DeveloperGuide.md]: Updated

File: docs/DeveloperGuide.md (hunk 3 above)
Comment (zengzihui) [reply docs/DeveloperGuide.md]: Changed

File: docs/DeveloperGuide.md (hunk 4 above)
Comment (zengzihui) [reply docs/DeveloperGuide.md]: removed

File: docs/DeveloperGuide.md (hunk 5 above)
Comment (zengzihui) [reply docs/DeveloperGuide.md]: remove alr

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -955,6 +955,39 @@ testers are expected to do more *exploratory* testing.
...
+       **Expected:** No buyer is added. Error indicating invalid format with constraints shown. <br><br>
+   
+    2. **Test case:** `addBuyer n/Kris Hua p/98765432 e/krishua@example.com budget/99999900` <br>
+       **Expected:** No buyer is added. Error indicating invalid command format. <br><br>
+
+3. **Invalid (Duplicate Person)**
+    1. **Test case:** `addBuyer n/John Carl 1 p/98765432 e/johncarl1@example.com budget/550000 type/Hdb`
+       **Expected:** No buyer is added. Error indicating that the person already existed in the data. <br><br>
+
+4. **Invalid Value for certain fields**
```
Comment (felixchanyy) [inline docs/DeveloperGuide.md:980]: Perhaps Invalid Value will do

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (KhoonSun47) [reply docs/DeveloperGuide.md]: This is a good fix.

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (KhoonSun47) [reply docs/DeveloperGuide.md]: This is a very good fix.

Comment (KhoonSun47) [reply docs/DeveloperGuide.md]: This is a good fix.

File: docs/DeveloperGuide.md (hunk 3 above)
Comment (KhoonSun47) [reply docs/DeveloperGuide.md]: This is a good fix.

File: docs/DeveloperGuide.md (hunk 4 above)
Comment (KhoonSun47) [reply docs/DeveloperGuide.md]: This is a good fix.

File: docs/DeveloperGuide.md (hunk 5 above)
Comment (KhoonSun47) [reply docs/DeveloperGuide.md]: This is a good fix.

File: docs/DeveloperGuide.md (hunk 6 above)
Comment (KhoonSun47) [reply docs/DeveloperGuide.md]: Yes I think the title needs to be changed.

**Review by felixchanyy — APPROVED** [review]
LGTM

**Review by redcolorbicycle — APPROVED** [review]

**Pull request merged**